pkg/cmd/pulumi/convert.go

```diff
@@ -149,6 +161,9 @@ func newConvertCmd() *cobra.Command {
 	cmd.PersistentFlags().BoolVar(
 		&strict, "strict", false, "If strict is set the conversion will fail on errors such as missing variables")
```

Comment:
```suggestion
		&strict, "strict", false, "Fail the on errors such as missing variables")
```

This is totally unrelated obviously, but this flag stood out a little compared to the help text on the other flags.  I figured I point it out anyway, but might be better to change in a separate PR?

Reply:
Can do yep :+1:

Reply:
"Fail the conversion on errors"